EventTicketingSystem.DataAccess/Interfaces/IEventInfoRepository.cs

```diff
@@ -0,0 +1,14 @@
+﻿using EventTicketingSystem.DataAccess.Models.Entities;
+
+namespace EventTicketingSystem.DataAccess.Interfaces;
+
+internal interface IEventInfoRepository
+{
+    Task<ICollection<EventInfo>> GetEvents();
+    Task<EventInfo> GetEventById(int id);
+    Task<ICollection<EventInfo>> GetEventsByCity(int cityId);
+    Task<ICollection<EventInfo>> GetEventsByVenue(int venueId);
+    Task AddEventInfo(EventInfo eventInfo);
```

Comment:
Looks like common CRUD operations can be extracted to the base interface

Reply:
Extracted interface IRepository<T>